pkg/apis/api.acorn.io/v1/computeclass.go

```diff
@@ -12,13 +12,29 @@ type ComputeClass struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 
-	Memory           v1.ComputeClassMemory        `json:"memory,omitempty"`
+	Memory           ComputeClassMemory           `json:"memory,omitempty"`
 	Resources        *corev1.ResourceRequirements `json:"resources,omitempty"`
 	Description      string                       `json:"description,omitempty"`
 	Default          bool                         `json:"default"`
 	SupportedRegions []string                     `json:"supportedRegions,omitempty"`
 }
 
+type ComputeClassMemory struct {
+	Min     string   `json:"min,omitempty"`
+	Max     string   `json:"max,omitempty"`
+	Default string   `json:"default,omitempty"`
+	Values  []string `json:"values,omitempty"`
+}
+
+func ComputeClassMemoryFromInternalAdmin(memory internaladminv1.ComputeClassMemory) ComputeClassMemory {
```

Comment:
Can you leave a comment here explaining *why* we're doing this so we don't forget in the future? What we're doing isn't entirely intuitive.